haverhill_311_function/app.py

```diff
@@ -17,8 +17,19 @@ def lambda_handler(event, context):
         Lambda Context runtime methods and attributes
     """
     data = qalert.pull_data_test()
-    qalert_requests: List[db.QAlertRequest] = sanitizer.sanitize(
-        qalert_data=data
-    )
-    with db.QAlertDB() as qalert_db:
-        qalert_db.save_many(requests=qalert_requests)
+    if data != []:
```

Comment:
better to return early:
```suggestion
    if not data:
        return
    # then normal execution
```